app/helpers/transcription_field_helper.rb

```diff
@@ -22,4 +24,35 @@ def field_layout(array)
     @field_array = array.zip(@values)
   end
 
+  def generate_field_input(field, cell)
+    input_name = formatted_field_name(field)
+
+    label = label_tag(field.label.parameterize, field.label.titleize)
```

Comment:
I titleize the label for the field inputs. Let me know if it should not be like this after all

Reply:
Let's not titleize the label -- many of our customers have strong opinions about this.